host/traceh.c

```diff
@@ -357,7 +381,7 @@ void oe_log_message(bool is_enclave, oe_log_level_t level, const char* message)
     strftime(time, sizeof(time), "%Y-%m-%dT%H:%M:%S%z", &t);
     long int usecs = tv.tv_usec;
 
-    if (!_initialized)
+    if (!_initialized_log_config)
     {
         initialize_log_config();
```

Comment:
In oe_set_host_log_level(), calling initialize_log_config() is protected by _log_lock. Here it's not protected.